src/executables/util_cs.c

```diff
@@ -3275,51 +3281,59 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
```

Comment:
I think error check is required

Reply:
`db_login()` prints error message. So we don't need to store the error message.